### DISCUSSION TITLE
add Formiz as an option of form library

### DISCUSSION THREAD

**Discussion by winner106** [blitz-js/blitz#3065] [Ideas]
### What do you want and why?

To create React forms, there is a lot of libraries out there. Formiz will help you create React forms with ease! Composable, headless & with built-in multi steps.

### Possible implementation(s)

[https://formiz-react.com/](https://formiz-react.com/)

**Comment by beerose** [comment]
Thanks for the suggestion!

I briefly went through Formiz docs, and it looks like it has its validation. That means we could no longer share Zod schemas between server and client. Is that right? Are you using it with Blitz and want to share how you're working with it in a Blitz project?

**Comment by winner106** [comment]
> Thanks for the suggestion!
> 
> 
> 
> I briefly went through Formiz docs, and it looks like it has its validation. That means we could no longer share Zod schemas between server and client. Is that right? Are you using it with Blitz and want to share how you're working with it in a Blitz project?
> 
> 
> 
> 

Yes , you are right. It has its validation.
I am not using formiz in a full stack project. So I have no idea how to working with it in a blitz project.....In my front end project ,formiz is a good react form,I hope someone knows how to use it in blitz project.

**Comment by beerose** [comment]
Hmm, I'll move it as a discussion then, since we don't know yet how it'll work with Blitz. Maybe other people provide some details. In the meantime, if you get this working, feel free to open a PR!
